src/server/controllers/users.controller.js

```diff
@@ -34,3 +12,15 @@ export function getAll(req, res, next) {
     }
   });
 }
+
+// Get all user list except logined user
+export function getMentorList(req, res, next) {
+  user.find({ 'email' : { $ne : 'session@yoda.com' }},  // if session_auth add, 'session' => req.session.email
+    { email: 0, userId: 0, age: 0, region: 0, skills: 0, regDate: 0 }, (err, doc) => {
```

Comment:
On line between 18 and 19, write a comment that says:

// TODO: Longer term, we should migrate to a UserSummary object 
// that contains subset of fields. For now, we return only the following 
// fields from user object:
//
// - email
// - userId
// - age
// - region
// - skills
// - regDate

This is probably the least you can do. You're writing the api, so this is just to help whoever that wants to use the user objects in other part of the system. 

I think in the longer term, we'd have to define UserSummary object that has only a subset of fields from User object, and get passed around the system. If someone wants more detailed fields for the user, they can do another db lookup. 

It's hairy :) that's why I initially suggested using the entire user object until the performance starts becoming an issue. But what you have here is ok for now.